modules/bim/game/tests/src/bim/game/factory/power_up.cpp

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// bim/modules/bim/game/tests/src/bim/game/factory/power_up.cpp
+#include <bim/game/arena.hpp>
+
+#include "bim/game/component/position_on_grid.hpp"
+#include <bim/game/component/bomb_power_up.hpp>
+#include <bim/game/component/flame_power_up.hpp>
+#include <bim/game/component/invisibility_power_up.hpp>
+
+#include <bim/game/factory/power_up.hpp>
+
+#include <entt/entity/registry.hpp>
+#include <entt/entt.hpp>
```

Comment:
We don't need to include the whole EnTT library here. Please remove this include directive.

In this file we only need `entt::registry` and `entt::entity`, which are both provided by `registry.hpp`.